pkg/imager/out.go

```diff
@@ -417,6 +417,14 @@ func (i *Imager) outInstaller(ctx context.Context, path string, report *reporter
 	newInstallerImg := mutate.MediaType(empty.Image, types.OCIManifestSchema1)
 	newInstallerImg = mutate.ConfigMediaType(newInstallerImg, types.OCIConfigJSON)
 
+	newInstallerImg, err = mutate.ConfigFile(newInstallerImg, &v1.ConfigFile{
```

Comment:
this is needed if we want to synthesize a proper multiarch manifest. Our previous base image was lacking this info